src/jbi/models.py

```diff
@@ -16,7 +17,8 @@ class Action(YamlModel):
     Action is the inner model for each action in the configuration file"""
 
     action: str = "src.jbi.whiteboard_actions.default"
-    contact: str
+    # TODO: Remove the tbd literal option when all actions have contact info
```

Comment:
#28 [lint 5/6] RUN pylint src tests
#28 5.251 ************* Module src.jbi.models
#28 5.251 src/jbi/models.py:20:5: W0511: TODO: Remove the tbd literal option when all actions have contact info (fixme)

Reply:
Resolved with your suggested commit below. Thanks!